packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -29,125 +29,31 @@ export const pageQuery = graphql`
-
-#### Brown Bag Session + Video Record on Abstract
-
-We'll document questions from the Abstract brown bag session that was hosted by our team, along with the video!
-
-### Process Updates
-
-#### Theme Switcher for Paste
-
-We'll polish the theme switching experience for Paste. This means that our implementation of SendGrid and Console themes will be more thorough.
-
-After this is released, if you see visual or functional issues in the doc site, let us know! It's a sign that we need to fix an underlying Paste component.
-
-</Box>
-<Box marginTop={60}>
+<Box>
 
 ## Release 0.4
 
-<Callout>
-  <CalloutText>
-    Codenamed <strong>that one after Oh! Point Three!</strong> This release is expected to land in the{' '}
-    <em>third week of November</em>. This release is <em>not locked in</em>, but here are candidates for the release.
-  </CalloutText>
-</Callout>
-
-- Polish and document Truncate component
-- Polish and document Media Object component
-- Ship our Grid component
-- Document accessibility guidelines and principles for Paste and downstream teams
-- Add svg-to-react as a library in Paste doc site
-- Build a List component
-- Design Input and Label component
-- Design a Tab component
-- Build doc site document outline system
-- Document our approach to writing CSS
-- Document our approach to building components
-- Document our approach to systemizing Twilio's design language
-- Design a Block Quote component for doc site
-- Cleaning up Tokens in SendGrid
-- Build a Props Table component for better documentation
-- Document our approach on responsiveness
+### Release Date: December 6, 2019
 
-## Release 0.5
+- Flex Tokens will be added to our Design System Sketch files
+- Theme Switcher will work as intended, allowing all styles to be viewable based on SendGrid and Console Themes.
+- Buttons will be productionized 🎉 and usable on SendGrid / Console.
+- Anchors will be productionized too! 🔗
+- Icon Systems will also be on the production train 🚝
+- Alerts will be design ready, and built in an upcoming release.
+- Grids will be design ready, and built in an upcoming release.
+- Form Inputs (Inputs and Textareas) will be design ready, and built in an upcoming release.
 
-<Callout>
-  <CalloutText>
-    We are working hard on a unique codename for this release. This release is expected to land in the{' '}
-    <em>first week of January, 2020</em>. This release is <em>not locked in</em>, but here are candidates for the
-    release.
-  </CalloutText>
-</Callout>
-
-- Improvements for visual regression testing on the website, and core features
-- Ship Icon System for the doc site
-- Ship Input and Label components
-- Design a Select component
-- Design a Form Footer component
-- Design Checkbox and Radio Button components
-- Implement continuous deployment of Paste packages
+## Release 0.5
 
-## Releases in 2020
+### Release Date: January 17, 2020
```

Comment:
Did you double-check this date?  I thought it was Jan 31 (12 weeks from yesterday)?